### PR TITLE
commercial: Add Lima-Labs

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

File: support/commercial.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@ Paid Support and Services
 
 Here are some commercial service and support providers for borgbackup:
 
+- `Lima-Labs <https://storage.lima-labs.com/>`_ - remote live file systems with native borg support
```
Comment (ThomasWaldmann) [inline support/commercial.rst:8]: move this to the end of the list please.

also, when users look at this list, guess they will ask themselves what's the difference between:
- `remote live file system` and `cloud storage platform`
- `native borg support` and `borg support`

In general, i want the least marketing phrasing here, just telling what they offer.

Comment (ThomasWaldmann) [reply support/commercial.rst:8]: maybe we should just say "storage with borg support".

Comment (DamienCassou) [reply support/commercial.rst]: I agree with you. Fixed

**Comment by DamienCassou** [comment]
I took the feedback into account.

File: support/commercial.rst (hunk 1 above)
Comment (ThomasWaldmann) [reply support/commercial.rst]: ok, thanks!

**Pull request merged**